jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java

```diff
@@ -86,15 +87,15 @@ private void testSourceCoverage(String fileName, TokenList tokens) {
             } else {
                 logger.info("Coverage: %.1f%%.".formatted(coverage * 100));
                 logger.info("Missing lines {}", codeLines);
-                if (coverage - 0.9 <= EPSILON) {
-                    // TODO use fail() instead when frontend is ready
+                if (coverage - BASELINE_COVERAGE <= EPSILON) {
                     logger.error("Source coverage is unsatisfactory");
+                    assertTrue(false);
```

Comment:
Closing this was maybe a bit too fast...
This should be `assertEquals("source coverage is only <format coverage> but should be <format expected coverage>", BASELINE_COVERAGE, coverage, EPSILON)`

Reply:
The method that you refer to checks that the absolute difference is smaller than a delta, right? `coverage` is supposed to be greater than or equal to `BASELINE_COVERAGE`.

Reply:
Good point. But the if-condition should still be inlined to the assert statement.
`assertTrue(<description>, coverage - BASELINE_COVERAGE > EPSILON)`

Reply:
Ah, much better, that's right. I pushed this version now. I need to open another PR, right?

Reply:
yes